libs/store/src/lib/loggers/race.ts

```diff
@@ -0,0 +1,16 @@
+import { AppErrorCode, AppMessageLogType } from '@razor/models';
+import { Dispatch } from '../store';
+
+export const raceNotFound = async (
+  dispatch: Dispatch,
+  rid: string,
+  additionalMessage?: string,
+) => {
+  dispatch.game.sendLogMessage({
```

Comment:
Do we need to `await` on the result?
If not, why do we have specified an `async` function?

Reply:
Fixed in PR #14 under commit 563c7da7291336f0b666f5c471e3d5046054c636